### ISSUE TITLE
Error with compiling from source

### ISSUE THREAD

**Issue by anuraj-rp** [ethz-asl/ethzasl_icp_mapping#14]
Hi,

I downloaded the source of the stack ethzasl_icp_mapping with
git clone --recursive https://github.com/ethz-asl/ethzasl_icp_mapping.git

and after running the rosdep commands mentioned on the ROS wiki here
http://www.ros.org/wiki/ethzasl_icp_mapping

I get an error which says the following:
/home/spacemaster/fuerte_workspace/sandbox/ethzasl_icp_mapping/ethzasl_icp_mapper/src/dynamic_mapper.cpp:14:39: fatal error: pointmatcher/MatchersImpl.h: No such file or directory

when I comment out the line #include "pointmatcher/MatchersImpl.h" in dynamic_mapper.cpp the source gets compiled. 

What does this header file do?? How important is that file for the functioning of the ROS stack. 

Basically I want to use the inspectors to test the performance. Can the inspectors be used with the MatchersImpl.h header file inlcluded?? 

NOTE: I face the same problem with "git clone --recursive git://github.com/ethz-asl/ethzasl_icp_mapping.git". There is difference in the files that get cloned by this command though.
